_posts/2019-09-11-how-to-stay-up-to-date.markdown

```diff
@@ -0,0 +1,52 @@
+---
+layout: post
+title: How to stay up to date with your Rails application
```

Comment:
@bronzdoc Please titleize using the same pattern we have been using in other articles. For example: How to Upgrade Any Rails Application Using Docker (see: https://fastruby.io/blog/rails/upgrades/southeast-ruby-upgrade-rails-workshop.html)